doc/classes/SpinBox.xml

```diff
@@ -55,6 +55,9 @@
 		<member name="suffix" type="String" setter="set_suffix" getter="get_suffix" default="&quot;&quot;">
 			Adds the specified [code]suffix[/code] string after the numerical value of the [SpinBox].
 		</member>
+		<member name="update_on_text_changed" type="bool" setter="set_update_on_text_changed" getter="get_update_on_text_changed" default="false">
+			Sets the value of the [Range] for this [SpinBox] when the [LineEdit] text is [code]changed[/code] instead of [code]submitted[/code]. See [signal LineEdit.text_changed] and [signal LineEdit.text_submitted]
```

Comment:
Since these don't refer to actual code identifiers (but only parts of them), I'd wouldn't use `[code]`, `[i]` might be better to emphasis. Also missing dot at EOL.
```suggestion
			Sets the value of the [Range] for this [SpinBox] when the [LineEdit] text is [i]changed[/i] instead of [i]submitted[/i]. See [signal LineEdit.text_changed] and [signal LineEdit.text_submitted].
```